brave/brave-core
# Issue 5690: Add key for brave services

Fix https://github.com/brave/brave-browser/issues/5690

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

1. Open Fiddler and enable SSL proxying: Settings > HTTPS > Decrypt HTTPS Traffic
2. Download the root cert and enable trust for the cert using `Keychain Access`
3. Verify that all requests to *.brave.com have the header `BraveServiceKey: dummytoken

Services:

- [x] crlsets.brave.com
- [ ] redirector.brave.com
- [x] go-updater.brave.com
- [x] static.brave.com
- [x] static1.brave.com
- [ ] clients4.brave.com
- [x] componentupdater.brave.com
- [ ] crxdownload.brave.com
- [ ] extensionupdater.brave.com
- [x] safebrowsing.brave.com
- [x] brave-core-ext.s3.brave.com

Remaining proxies are currently blocked on: https://github.com/brave/brave-browser/issues/6331

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
